src/Components/settings-dropdown/email-configuration.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { useState } from "react";
+
+import "./settings.css";
+import {
+  testEmailSettings,
+  saveEmailSettings,
+} from "../../services/email-settings-service";
+
+const EmailConfiguration: React.FC = () => {
+  // const api_url = process.env.REACT_APP_API_URL || "";
+  const [server, setServer] = useState<string>("");
+  const [port, setPort] = useState<string>("");
+  const [username, setUsername] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
+  const [securityType, setSecurityType] = useState<string>("");
+  const [testEmail, setTestEmail] = useState<string>("");
+  const [organizationId, setOrganizationId] = useState<string>(
+    "669f03102a97b116272c2085"
```

Comment:
**Actionable**
Can you add a `// @todo` comment as to why we hardcoded the string and what to do to fix this later?

The idea behind the comment is to let others know that you've consciously left this hardcoded value here and provide context with a potential solution.

For example:
```
// Hardcoded organization id because we're wrapping up the auth process. For testing purposes: replace value with you org id. 
// @todo Replace hardcoded orgId during/after auth integration.
```